packages/loader/container-loader/src/container.ts

```diff
@@ -169,14 +170,15 @@ export enum ConnectionState {
  */
 export async function waitContainerToCatchUp(container: IContainer) {
     // Make sure we stop waiting if container is closed.
+    const containerClosedMessage = "Container is closed";
     if (container.closed) {
-        throw new Error("Container is closed");
+        throw new Error(containerClosedMessage);
```

Comment:
Can you switch this to new UsageError? Indicating it's essentially not valid to call this function with a closed container.